### ISSUE TITLE
latest release of nodemod v2.7.1 breaks datepicker

### ISSUE THREAD

**Issue by zenwork** [motss/app-datepicker#197]
### Description
dependency on nodemod@^2.6.1 pulls in version 2.7.0 which results in export errors when using rollup and node-resolve plugin

### Expected outcome
app-datepicker works correctly

### Actual outcome
```
Could not resolve import "nodemod/dist/calendar/helpers/to-utc-date.js" 
in "/Users/flo/Dev/work/alpha/niu/node_modules/app-datepicker/dist/datepicker.js". 
Package subpath "./dist/calendar/helpers/to-utc-date.js" 
is not defined by "exports" in /Users/flo/Dev/work/alpha/niu/node_modules/nodemod/package.json

  Error: Could not resolve import "nodemod/dist/calendar/helpers/to-utc-date.js".
      at Object.resolveImport (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-rollup/dist/rollupAdapter.js:118:35)
      at async Object.resolve (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-rollup/dist/createRollupPluginContextAdapter.js:52:36)
      at async Object.resolveImport (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-rollup/dist/rollupAdapter.js:94:30)
      at async resolveImport (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/plugins/transformModuleImportsPlugin.js:126:30)
      at async transformImport (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/plugins/transformModuleImportsPlugin.js:139:37)
      at async maybeResolveImport (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/plugins/transformModuleImportsPlugin.js:68:41)
      at async transformImports (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/plugins/transformModuleImportsPlugin.js:93:36)
      at async Object.transform (/Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/plugins/transformModuleImportsPlugin.js:170:49)
      at async /Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/middleware/pluginTransformMiddleware.js:44:32
      at async /Users/flo/Dev/work/alpha/niu/node_modules/@web/dev-server-core/dist/middleware/pluginFileParsedMiddleware.js:15:9
```

### Steps to reproduce
1. Put a `app-datepicker` element in the page.
2. bundle app with rollup and node-resolve plugin

my rolup config:
``` js
import commonjs        from '@rollup/plugin-commonjs'
import { nodeResolve } from '@rollup/plugin-node-resolve'
import typescript      from '@rollup/plugin-typescript'
import filesize        from 'rollup-plugin-filesize'
import sourcemaps      from 'rollup-plugin-sourcemaps'
import postcss from 'rollup-plugin-postcss'


export const libConfig= (isProd) =>{
  return {
    input: './index.ts',
    output: [
      { dir: 'dist', format: 'es', sourcemap: true, entryFileNames: '[name].js', chunkFileNames: '[name]-chunk.js' }
    ],
    plugins: [
      typescript(),
      nodeResolve(),
      postcss({
        extensions: ['.css'],
        modules: true
      }),
      commonjs(),
      sourcemaps(),
      ...isProd? [filesize()] : []
    ]
  }
}
```

### Workaround
If I force the dependency to be on 2.6.1 by adding a dependency in my project then the bundling error goes away.

**Comment by motss** [comment]
There is no v2.7.1. Let me look into that.

**Comment by motss** [comment]
Can you reinstall again? The latest version is now `v2.7.1`.

I've also tried to reproduce the error but failed to get the error message you're experiencing. Do you mind creating a reduced test case in a new Github repo so that I can clone it and run it locally.

**Comment by motss** [comment]
@zenwork Fixed in 0270ac03b79e8e57b2b2692541c03796a05fd77a.
